src/views/Receiver.vue

```diff
@@ -130,6 +130,11 @@ import Player from '@/components/Player/Player.vue';
 
 export default {
   name: 'Receiver',
+  metaInfo() {
+    return {
+      title: `2N.fm - ${this.roomName}(${this.receiverViewerCount + 1})`,
```

Comment:
Just a question on the copy: should this be `${this.roomName} (${this.receiverViewerCount + 1 })` -- i.e. literally just add a space in between the room_id and the viewer count's parenthesis?

Same would apply to Streamer.vue if so.